src/Task/Git/CommitMessage.php

```diff
@@ -96,7 +96,18 @@ public function run(ContextInterface $context)
         }
 
         if (count($exceptions)) {
-            return TaskResult::createFailed($this, $context, implode(PHP_EOL, $exceptions));
+            return TaskResult::createFailed(
+                $this,
+                $context,
+                implode(PHP_EOL, $exceptions) . <<<KAWAii
```

Comment:
What does KaWaii even mean?! Maybe you could give the heredoc a more generic name?
Normally this kind of formatting is added to a formatter. That way you can change it to whatever formatting you like. However, the current formatters implementation is based on the processes. Maybe a `ContextFormatterInterface` should also be added for cases like this.